ramls/instance.json

```diff
@@ -346,6 +346,15 @@
       "folio:linkFromField": "id",
       "folio:linkToField": "instanceId",
       "folio:includedElement": "holdingsRecords"
+    },
+    "natureOfContentTermIds": {
```

Comment:
Hi @mreno-EBSCO,

Thanks for pointing on this. Yep, you're right, fixed now.

Not sure about case sensitivity, I think UI should provide proper NatureOfContentTerm.ID and we just need to check that no duplicates has been specified + the UUID string has the proper format. 

Is it make sense?

Thanks,
Bohdan

Reply:
@mreno-EBSCO Do any of the systems in FOLIO check for unique items in an array, or is it just for informational purposes for the client?

Reply:
I believe a Set is generated instead of a List. I don't know if there is any validation done, like a 422 when there are duplicates. I assume they are just coalesced via the Set.

Reply:
@mreno-EBSCO @bohdan-suprun  Is this important enough that we want to test the behaviour? For example, we submit an instance representation with two references to the same nature of content term and receive back a representation with only a single reference, or does it error?

Reply:
@marcjohnson-kint I think, if this is implemented (I guess it was), then we should have a test for it in case something unexpected happens in a new version of RMB. There have been cases where devs forgot or removed the `@Validate` annotation from the impl... hold on... I just looked, and the annotation is not present on any method in `InstanceStorageImpl.java`. I don't think any of the aspectj based validation is being performed here. The `validate` trait is not present in the raml as well. It is there for `item-storage`. Hmm... ah-ha... found some familiar JIRAs:

This JIRA brings back memories: [MODINVSTOR-24](https://issues.folio.org/browse/MODINVSTOR-24). I created it almost 2 years ago. 🙄
You also created an issue, Marc: [MODINVSTOR-94](https://issues.folio.org/browse/MODINVSTOR-94)

Maybe we should add this to the tech debt list for an upcoming sprint.

Reply:
Actually, I consider this a bug and not tech debt. We should really schedule it before something bad gets to the DB level. Maybe I am a bit too paranoid about the lack of validation. 😄

Reply:
> Actually, I consider this a bug and not tech debt. We should really schedule it before something bad gets to the DB level. Maybe I am a bit too paranoid about the lack of validation. 😄

My understanding of FOLIO technical debt is that it can be any issue type. Please feel free to include it in the tech debt options for Cate in the next few weeks.

We need to be aware that it is a breaking implementation compatibility change, and so might cause unexpected issues in the UI.